go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/docker/docker v20.10.7+incompatible
 	github.com/docker/go-connections v0.4.0 // indirect
 	github.com/fatih/color v1.9.0 // indirect
+	github.com/golang/protobuf v1.5.2 // indirect
```

Comment:
This is the old protobuf library; we should not depend on this. Check your code for imports using `github.com/golang/protobuf` and replace it with `google.golang.org/protobuf`.

Reply:
My outdated protoc-gen-go imported this in ag.pb.go... Fixed it now.